internal/integration/rest/models.go

```diff
@@ -63,6 +63,13 @@ type (
 		ParendID int    `json:"parentId"`
 	}
 
+	Project struct {
+		ID       int    `json:"id"`
+		TeamID   int    `json:"teamId"`
+		Name     string `json:"name"`
+		IsPublic bool   `json:"isPublic"`
```

Comment:
Please confirm with Wombat team these are the fields they need for project import in AST.